lib/index.ts

```diff
@@ -135,6 +135,8 @@ export async function need(opts: NeedOptions) {
     }
   }
 
+  throw new Error("Could not find binaries in PKG_CACHE_PATH nor default USER/.pkg-cache. Amplify's fork requires that cache with binaries is prepared before executing this step.");
```

Comment:
Why is this error here rather than after line 134?

Reply:
In case `forceBuild` is set to true. We want to skip logic below in any case.

Reply:
We want it to be outside that if statement so that the error is thrown in the event that the if statement is not entered.

Reply:
I might be misreading the indentation, but doesn't this make line 140 unreachable?

Reply:
Yes, that's the intention.
(plus the fact that we want to keep codebase as close to upstream as possible).